lib/iris/coords.py

```diff
@@ -1109,6 +1109,11 @@ def cell(self, index):
                 point = self.units.num2date(point)
                 if bound is not None:
                     bound = self.units.num2date(bound)
+                    
+        else
+            warn_deprecated("the 'Future' object property "
+                            "cell_datetime_objects is now deprecated. Please "
+                            "remove code which uses this. cell_datetime_objects")
```

Comment:
This has gone a bit weird.  You need to re-order the deprecation warning message as it has got the words and things all mixed up.